test/elm/parameters/test.coffee

```diff
@@ -171,6 +171,29 @@ describe 'DateTimeParameterTypes', ->
     d = "2012-10-25T12:55:14.456+00"
     should(() => @foo2.exec(@ctx.withParameters { FooP: d })).throw(/.*wrong type.*/)
 
+describe 'DateParameterTypes', ->
+  @beforeEach ->
+    setup @, data
+
+  it 'should execute to provided valid value', ->
+    d = Date.parse('2012-10-25')
+    @foo.exec(@ctx.withParameters { FooP: d }).should.equal d
+
+  it 'should throw when provided value is wrong type', ->
+    d = "2012-10-25"
+    should(() => @foo.exec(@ctx.withParameters { FooP: d })).throw(/.*wrong type.*/)
+
+  it 'should execute to default value', ->
+    @foo2.exec(@ctx).should.eql Date.parse('2012-04-01')
```

Comment:
This test is failing. Looks like it is expecting a DateTime rather than a date. Which is odd because the expected value here is coming from Date.parse which does return a Date

Reply:
Correction: it is the DateTimeParameterTypes test, not the DateParameterTypes test